SWLOR.Game.Server/Feature/ChatCommandDefinition/SystemChatCommand.cs

```diff
@@ -45,105 +44,14 @@ private static void BugCommand(ChatCommandBuilder builder)
                         {
                             return "You may only submit one bug report per minute. Please wait and try again.";
                         }
+                        SetLocalString(user, "BUG_REPORT_LAST_SUBMISSION", dateTime.ToString());
```

Comment:
It might make more sense to do this check when they submit on the UI. As it is now, if they run this command the timer starts as soon as they enter the command. If the user decides to cancel they need to wait a minute before they can open it again. I'd probably get rid of this validation section and do it on the UI's view model for the OnClick event. What do you think?

Reply:
Good call, it was actually being set in OnClick event already anyway.

Reply:
Done